packages/imitation_game/tests/smiley/flutter/smiley/linux/CMakeLists.txt

```diff
@@ -0,0 +1,95 @@
+cmake_minimum_required(VERSION 3.10)
```

Comment:
maybe add linux later if we end up using it?

Reply:
Sure, I don't know how easy it is to add later, removed.